Linkta-org/core
# Docs: Cleanup Docs Folder

## Description

- Moved `/docs` to the root directory
- Renamed files to maintain a consistent naming style
- Reorganized the folder structure for improved maintainability
![image](https://github.com/user-attachments/assets/4e4f02cb-f22c-40ab-bb87-c20dac74bdaf)
- Removed sprint planning file as it is no longer in use

Fixes #NA

## Type of change


## How Can this be tested? Testing Plan to review this PR

I have manually tested all the reference links to ensure they all work after the name change.

## Checklist:
